src/tests/multihost/alltests/test_krb_ldap_connection.py

```diff
@@ -270,6 +270,8 @@ def run_test(self, timeout, multihost):
         assert cmd.returncode == 0
 
         def find_local_port():
+            netstat_pkg = 'dnf install -y net-tools'
+            multihost.client[0].run_command(netstat_pkg)
```

Comment:
Hi,

wouldn't it be more consistent to add `net-tools` to `client_install_pkgs` in `./src/tests/multihost/sssd/testlib/common/utils.py` ?

bye,
Sumit

Reply:
Hi Sumit,

Yes, sending an updated patch

Reply:
It might be better to use ss instead of netstat as it is deprecated and it might disappear completely at some point.

Reply:
Good point. Replaced netstat with ss.